Modules/posixmodule.c

```diff
@@ -1110,6 +1109,15 @@ get_posix_state(PyObject *module)
  * Input fields:
  *   path.nullable
  *     If nonzero, the path is permitted to be None.
+ *   path.nonstrict
+ *     If nonzero, the path is permitted to contain
+ *     embedded null characters and have any length.
+ *   path.make_wide
+ *     If nonzero, the converter always uses wide, decoding if necessary, else
+ *     it always uses narrow, encoding if necessary. The default value is
+ *     nonzero on Windows, else zero.
+ *   path.suppress_value_error
+ *     If nonzero, raising ValueError is suppressed.
```

Comment:
Why not any exception? Thinking Unicode errors specifically. (I assume this was discussed earlier but haven't read all the past comments.)

Reply:
Value errors suffice at the moment: `class UnicodeError(ValueError): ...`.

- https://github.com/python/cpython/pull/118355#issuecomment-2093086005
- https://github.com/python/cpython/pull/118355#discussion_r1590006522